lib/omise/charge.ex

```diff
@@ -254,12 +257,21 @@ defmodule Omise.Charge do
   ## Examples
 
       Omise.Charge.capture("chrg_test_4xso2s8ivdej29pqnhz")
+      Omise.Charge.capture("chrg_test_4xso2s8ivdej29pqnhz",key: "skey_test_123")
+      Omise.Charge.capture("chrg_test_4xso2s8ivdej29pqnhz",[capture_amount: 3000])
+      Omise.Charge.capture("chrg_test_4xso2s8ivdej29pqnhz",[key: "skey_test_123"],[capture_amount: 3000])
 
   """
-  @spec capture(String.t(), Keyword.t()) :: {:ok, t} | {:error, Omise.Error.t()}
-  def capture(id, opts \\ []) do
-    opts = Keyword.merge(opts, as: %__MODULE__{})
-    post("#{@endpoint}/#{id}/capture", [], opts)
+  @spec capture(String.t(), Keyword.t(), Keyword.t()) :: {:ok, t} | {:error, Omise.Error.t()}
+  def capture(id, opts \\ [], params \\ []) do
```

Comment:
Looking at the rest of the module, it seems that `params` usually comes before `opts` in the argument list. In this case, so as not to break the API, I would keep the existing function as it is and add a 3-arity function (with no default values) for the case where `params` are needed. Here is an example:

```
@doc """
Capture a charge with additional parameters.
"""
@spec capture(String.t(), Keyword.t(), Keyword.t()) :: {:ok, t} | {:error, Omise.Error.t()}
def capture(id, params, opts) do
  post("#{@endpoint}/#{id}/capture", params, opts)
end
```